tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<ul><li>The way you put it, it seems like the only things we can change is <code>rydberg_level</code> and <code>reusable_channels</code>, which is not the case. </li><li>You should hide the link in an hyperlink   </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>